### DISCUSSION TITLE
Can't link companies to Freshbooks

### DISCUSSION THREAD

**Discussion by dronsfield** [codatio/codat-docs#249] [Q&A] [answered]
Hello Codat,

I'm wondering if I can get a little help. I can't seem to link any companies to Freshbooks after setting up the integration. 

Thanks,
Josh

**Comment by d-laing** [comment, marked as answer]
Hello Josh,

Sure. First make sure you've selected all the necessary scopes on your FreshBooks App in the FreshBooks Developer Portal. Instructions on how to do that are [here](https://docs.codat.io/integrations/accounting/freshbooks/accounting-freshbooks-setup#create-a-freshbooks-application) and [here](https://docs.codat.io/integrations/accounting/freshbooks/accounting-freshbooks-setup#freshbooks-application-scopes).

If you need to push data to FreshBooks, make sure you select the **write** scopes.

Hope that helps!

Daniel
Technical Writer, Codat
